src/portable/renesas/rusb2/dcd_rusb2.c

```diff
@@ -266,6 +266,13 @@ static void pipe_read_packet_ff(rusb2_reg_t * rusb, tu_fifo_t *f, volatile void
   tu_fifo_advance_write_pointer(f, count);
 }
 
+
+static void wait_pipe_fifo_empty(rusb2_reg_t* rusb, uint8_t num) {
```

Comment:
Because of `TU_ASSERT` returns `bool` value, this function's return type should be `bool`. 
Please find the build error message.
https://github.com/hathach/tinyusb/actions/runs/6119698047/job/16610133014?pr=2245#step:10:45